test/article/model.js

```diff
@@ -1,58 +0,0 @@
-/**
- * Module dependencies.
- */
-var should = require('should'),
```

Comment:
Why delete this file?

Reply:
I briefly removed the file to test out how the `article` worked in the application, but  I didn't recommit it back before making this push.
It would be added back to my next push.